core/iwasm/fast-jit/fe/jit_emit_control.c

```diff
@@ -601,7 +655,7 @@ bool
 jit_compile_op_block(JitCompContext *cc, uint8 **p_frame_ip,
                      uint8 *frame_ip_end, uint32 label_type, uint32 param_count,
                      uint8 *param_types, uint32 result_count,
-                     uint8 *result_types)
+                     uint8 *result_types, bool merge_cmp_and_if)
```

Comment:
I thought the merge action is by default. Why need to disable it ?

Reply:
For the float/double comparison opcodes, we might translate them into calling native function instead, but not `CMP + SELECTcc`

Reply:
In that case, shall we tell that by checking `JIT_REG_KIND_XXX`?